frontend/src/lib/notify/notify.svelte

```diff
@@ -0,0 +1,23 @@
+<script lang="ts">
+  import { notifications, removeAllNotifications, removeNotification} from './index.js';
+</script>
+
+<div class="toast toast-center w-96">
```

Comment:
What's the `w-96` about?

Reply:
Without it this happens:
![image](https://github.com/sillsdev/languageforge-lexbox/assets/61605641/a8dae9d2-343b-4c77-b0e5-aef8d4c936d5)

Reply:
Sounds good 😃